Task 3/Task 3.3/Task 3.3/Task 3.3/ExtensionMethods.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Task_3._3
+{
+    public static class ExtensionMethods
+    {
+        public static void CheckLanguage(this string text)
+        {
+            Regex english = new Regex(@"^\W*([a-zA-z]+\W*\s*)+$");
+            Regex russian = new Regex(@"^\W*([а-яёА-ЯЁ]+\W*\s*)+$");
+            Regex digits = new Regex(@"^\W*([0-9]+\W*\s*)+$");
+            if (english.IsMatch(text))
+            {
+                Console.WriteLine("English");
+            }
+            else if (russian.IsMatch(text))
+            {
+                Console.WriteLine("Russian");
+            }
+            else if (digits.IsMatch(text))
+            {
+                Console.WriteLine("Digits");
+            }
+            else 
+            {
+                Console.WriteLine("Mixed");
+            }
```

Comment:
This method should be pure so we can actually use its findings. (stop using `Console` in purely logic methods/classes)